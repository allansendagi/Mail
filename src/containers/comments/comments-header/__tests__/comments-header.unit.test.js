import React from 'react';
import {shallow, configure} from 'enzyme';
import {CommentsHeader} from '../comments-header.component';
import Adapter from 'enzyme-adapter-react-16';

// configure({adapter: new Adapter()});
describe('CommentsHeader', () => {
  test('CommentsHeader renders with props.amountComments = null', () => {
    const wrapper = shallow(
      <CommentsHeader/>
    );
    expect(wrapper).toMatchSnapshot();
  });

  test('CommentsHeader renders with props.amountComments = 0', () => {
    const wrapper = shallow(
      <CommentsHeader amountComments={123}/>
    );
    expect(wrapper).toMatchSnapshot();
  });
});