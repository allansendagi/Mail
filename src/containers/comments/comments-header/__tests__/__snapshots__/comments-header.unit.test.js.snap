// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommentsHeader CommentsHeader renders with props.amountComments = 0 1`] = `ShallowWrapper {}`;

exports[`CommentsHeader CommentsHeader renders with props.amountComments = null 1`] = `ShallowWrapper {}`;
